ParseUI/Classes/LogInViewController/PFLogInView.m

```diff
@@ -35,6 +35,16 @@
 static NSString *const PFLogInViewDefaultFacebookButtonImageName = @"facebook_icon.png";
 static NSString *const PFLogInViewDefaultTwitterButtonImageName = @"twitter_icon.png";
 
+#pragma mark - Accessibility labels identifiers
```

Comment:
Nit: Use common style for pragma marks. This makes them waaaay more visible in text editor itself, without using `Document Items` menu in Xcode.
```
///--------------------------------------
#pragma mark - Accessibility Identifiers
///--------------------------------------
```